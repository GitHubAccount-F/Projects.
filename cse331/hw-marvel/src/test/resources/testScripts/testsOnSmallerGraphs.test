#Tests my algorithms with the example csv given

#Creates graph
LoadGraph graph1 staffSuperheroes.csv

#Finds paths
FindPath graph1 Ernst-the-Bicycling-Wizard Grossman-the-Youngest-of-them-all

FindPath graph1 Perkins-the-Magical-Singing-Instructor Ernst-the-Bicycling-Wizard