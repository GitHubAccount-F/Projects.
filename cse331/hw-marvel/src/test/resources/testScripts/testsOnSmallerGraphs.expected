#Tests my algorithms with the example csv given

#Creates graph
loaded graph graph1

#Finds paths
path from Ernst-the-Bicycling-Wizard to Grossman-the-Youngest-of-them-all:
Ernst-the-Bicycling-Wizard to Grossman-the-Youngest-of-them-all via CSE331

path from Perkins-the-Magical-Singing-Instructor to Ernst-the-Bicycling-Wizard:
Perkins-the-Magical-Singing-Instructor to Ernst-the-Bicycling-Wizard via CSE331